src/components/MarqueeSkills.tsx

```diff
@@ -0,0 +1,53 @@
+import { SKILLS } from '@/data/skills';
+import marqueeStyles from '@/styles/marquee.module.css';
+
+interface MarqueeSkillsProp {
+    direction: 'forward' | 'reverse';
+}
+
+/** Shows the Skills using the Marquee Effect. */
+export function MarqueeSkills({ direction }: MarqueeSkillsProp) {
+    return (
```

Comment:
Nit: we can provide a default value to the prop. and accordingly we can change the ternary condition as well.

```suggestion
export function MarqueeSkills({ direction = 'forward' }: MarqueeSkillsProp) {
    return (
```

Reply:
Made the `direction` prop optional with default value.
https://github.com/dhruvdhaduk-simform/portfolio/pull/2/commits/52dcc7402c7fbbea2f719dcfa82ad035e9c0a254